dali/operators/operators.cc

```diff
@@ -13,8 +13,18 @@
 // limitations under the License.
 
 #include "dali/core/api_helper.h"
-#include "dali/operators/operators.h"
+#include "dali/operators.h"
+
+/*
+ * The point of these functions is to force the linker to link against dali_operators lib
+ * and not optimize-out symbols from dali_operators
```

Comment:
You are not linking statically. So this comment is not true. I think the bellow comment is more accurate.

Reply:
Ummm, even linking dynamically, I do need to call these functions for this particular reason. If I wouldn't call them, symbols would be optimized-out.

I put here both comments, cause the one above is better to understand and the one below is more accurate, as you say. I see no harm in having both ;)